package/libpiksi/libpiksi/src/settings.c

```diff
@@ -816,26 +781,16 @@ static int setting_read_watched_value(settings_ctx_t *ctx, setting_data_t *setti
     return -1;
   }
 
-  result =  setting_perform_request_reply_from(ctx,
-                                               SBP_MSG_SETTINGS_READ_REQ,
-                                               msg,
-                                               msg_len,
-                                               msg_len,
-                                               WATCH_INIT_TIMEOUT_MS,
-                                               WATCH_INIT_TRIES,
-                                               SBP_SENDER_ID);
+  result = setting_perform_request_reply_from(
+    ctx, SBP_MSG_SETTINGS_READ_REQ, msg, msg_len, msg_len, WATCH_INIT_TIMEOUT_MS, WATCH_INIT_TRIES, SBP_SENDER_ID);
```

Comment:
There's gotta be a setting for this... right?  Unfolding these doesn't not improve the readability IMO.